tests/test_tokenization_common.py

```diff
@@ -1153,6 +1153,39 @@ def test_chat_template_batched(self):
                     dummy_conversations, chat_template=dummy_template, tokenize=True
                 )  # Check that no error raised
 
+    @require_jinja
+    def test_jinja_extensions(self):
```

Comment:
This test name is far too generic! Is it testing extensions can be used? testing specific extensions? which extensions?

Reply:
Actually, even better, split it into two tests with clearer names!